.github/workflows/yml.yml

```diff
@@ -0,0 +1,18 @@
+name: YAML check
+
+on:
+  pull_request:
+    branches: [main]
+    paths:
+      - "**/*.yml"
+      - "**/*.yaml"
+      - ".github/workflows/yml.yml"
```

Comment:
Out of interest, why are these two explicitly required?

Reply:
Just so the (non required) workflow only triggers when modifying .yml/.yaml files. I suppose the workflow is yml too, so no need to specify it, I really added it out of habit without thinking this through 😄